WebDriverAgentLib/Utilities/FBClassChainQueryParser.m

```diff
@@ -0,0 +1,371 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "FBExceptionHandler.h"
+#import "FBClassChainQueryParser.h"
+#import "FBErrorBuilder.h"
+#import "FBElementTypeTransformer.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface Token : NSObject
```

Comment:
You need `FB` prefix in all of your classes. Also I would be more precise and name it something like  `FBClassChainToken` ?